### DISCUSSION TITLE
Qué se rellena el campo Mi matriz en el bot PBEMAIL

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#231] [2. Questions and Answers] [answered]
Hola

Vi que hay un nuevo bot de email, me costó un poco entender los campos que llenaba, pero me di cuenta que es el datos de la versión antigua.

Sin embargo el campo mi matriz, no sé para qué sirve.

Adjunto SS

![image](https://user-images.githubusercontent.com/61419471/120720000-4913b200-c499-11eb-9825-60c19f1696aa.png)

Gracias :>

**Comment by guillevalin** [comment, marked as answer]
@NicolasPLG Ese campo no debería existir, será eliminado en un hotfix que saldrá lo más pronto posible.

**Reply by NicolasPLG** [reply]
gracias @guillevalin
